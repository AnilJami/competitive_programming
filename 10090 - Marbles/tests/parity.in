1
2
8
10
21
1000000000
2147483646
2147483647
0