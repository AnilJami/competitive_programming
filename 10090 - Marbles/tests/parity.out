The parity of 1 is 1 (mod 2).
The parity of 10 is 1 (mod 2).
The parity of 1000 is 1 (mod 2).
The parity of 1010 is 2 (mod 2).
The parity of 10101 is 3 (mod 2).
The parity of 111011100110101100101000000000 is 13 (mod 2).
The parity of 1111111111111111111111111111110 is 30 (mod 2).
The parity of 1111111111111111111111111111111 is 31 (mod 2).